### DISCUSSION TITLE
Не удается отправить оценку статьи

### DISCUSSION THREAD

**Discussion by RoNUII** [doka-guide/platform#961] [Разное]
При нажатии кнопок для оценки **какой-либо** статьи:
![image](https://user-images.githubusercontent.com/49837634/180307530-4527a586-f36e-4f33-b889-b4f1a4f41273.png)

Консоль:
`POST https://api.doka.guide/form 401`
`Response {type: 'cors', url: 'https://api.doka.guide/form', redirected: false, status: 401, ok: false, …}`

Почему так, как быть и у одного меня ли такая ситуация?

**Comment by solarrust** [comment]
Привет! Спасибо за репорт! Переведу дискуссию в задачу в репозитории платформы. Обязательно посмотрим в чём проблема и починим 👍
